MPSGameLift/Documentation/GameLift.md

```diff
@@ -38,6 +38,16 @@ This README covers optional setup, testing and running on [Amazon GameLift](http
     ---
 It's important to make sure that the bootstrap.game.profile.setreg file has been added to one of the seed files. (also add debug if you want to support debug builds)
 
+---
+**NOTE**
+
+This will not build the game client. If you need the client for testing, also run this command:
+```
+cmake --build build\windows_mono --target MultiplayerSample.GameLauncher --config profile -- /m /nologo
```

Comment:
I would add context that the export_gamelift_server_package sets up to build things monolithic etc

Also could we add this as an option ie -build-test-client = true? to that script so it just does it?

Reply:
Yes, we discussed having an option with Gene. If we want to do that, we can spin up a separate task for it and either of us can pick it up.